### PR TITLE
feat(spotify): show whole artists of song

### PR DESCRIPTION
**Pull request by Banonevar**
###### CHECKLIST

- [ ] tests are changed or added
- [ ] documentation is changed or added
- [ ] locales are changed or added
- [ ] db relationship (tools/database) are changed or added
- [ ] commit message follows [commit guidelines](https://github.com/sogehige/sogeBot/blob/master/CONTRIBUTING.md#commit-guidelines)

### CODE REVIEW

File: src/bot/integrations/spotify.ts [hunk 1]
```diff
@@ -284,7 +284,7 @@ class Spotify extends Integration {
         currentSong = {
           finished_at: Date.now() - data.body.item.duration_ms, // may be off ~10s, but its important for requests
           song: data.body.item.name,
-          artist: data.body.item.artists[0].name,
+          artist: data.body.item.artists.map(o => o.name).join(', '),
```
[the remark this answers is not in the document]
Comment (Banonevar) [reply src/bot/integrations/spotify.ts]: maybe i'll call it **artists** then?

[the remark this answers is not in the document]
Comment (sogehige) [reply src/bot/integrations/spotify.ts]: sure, why not

[the remark this answers is not in the document]
Comment (Banonevar) [reply src/bot/integrations/spotify.ts]: then check my new commits, hope it's ok

**Comment by sogehige** [comment]
Still i cannot do anything with travis, do you properly rebased?

**Comment by Banonevar** [comment]
How i can do that `rebase`?

**Comment by sogehige** [comment]
`git pull -r upstream next; git push -f` ? Not sure never rebased pr from diffrerent git repo

**Comment by Banonevar** [comment]
yea i did same before creating PR i guess, but can recheck

**Comment by Banonevar** [comment]
I was right, i did everything rightly
img

**Comment by sogehige** [comment]
You can add commit with `[skip-tests]` in description to skip tests

File: src/bot/integrations/spotify.ts [hunk 2]
```diff
@@ -29,6 +29,7 @@ class Spotify extends Integration {
     uri: string;
     requestBy: string;
     artist: string;
+    artists: string,
```
Comment (sogehige) [inline src/bot/integrations/spotify.ts]: => `artists: string;`

Comment (Banonevar) [reply src/bot/integrations/spotify.ts]: I already did this, check `upd` commit

Comment (Banonevar) [reply src/bot/integrations/spotify.ts]: nah, you mean fix `,` by `;`, ok

**Pull request merged**